src/bors/handlers/review.rs

```diff
@@ -52,6 +55,29 @@ pub(super) async fn command_unapprove(
     notify_of_unapproval(&repo_state, pr).await
 }
 
+pub(super) async fn handle_pull_request_edited(
+    repo_state: Arc<RepositoryState>,
+    db: Arc<PgDbClient>,
+    payload: PullRequestEdited,
+) -> anyhow::Result<()> {
+    // If the base branch has changed, unapprove the PR
+    let Some(_) = payload.from_base_sha else {
+        return Ok(());
+    };
+
+    let pr_model = db
+        .get_or_create_pull_request(repo_state.repository(), payload.pull_request.number)
+        .await?;
+    if pr_model.approved_by.is_none() {
```

Comment:
We will probably ask this question in more places, so it would be good to add a more high-level query for this (simply `pr_model.is_approved()`. We can do that in some follow-up PR.